pkg/minishift/oc/oc.go

```diff
@@ -18,17 +18,19 @@ package oc
 
 import (
 	"fmt"
+
 	"github.com/minishift/minishift/pkg/minikube/constants"
 	"github.com/minishift/minishift/pkg/util"
 
 	"bytes"
-	"github.com/minishift/minishift/pkg/util/cmd"
-	"github.com/minishift/minishift/pkg/util/filehelper"
-	"github.com/pkg/errors"
 	"io"
 	"os"
 	"regexp"
 	"strings"
+
+	"github.com/minishift/minishift/pkg/util/cmd"
+	"github.com/minishift/minishift/pkg/util/filehelper"
+	"github.com/pkg/errors"
```

Comment:
@anjannath these are left over changes from some older change.  Please remove these.

Reply:
Sorry for the delay, i did not see this comment, doing now.